src/hooks/useAutoHeight.test.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
The file we are testing against is a hook which would be a `ts` file.
I would also expect this file to be a `ts` file.

Reply:
Converted to `.ts`